lib/cosmos/interfaces/serial_interface.rb

```diff
@@ -61,9 +62,20 @@ def connect
         @parity,
         @stop_bits,
         @write_timeout,
-        @read_timeout
+        @read_timeout,
+        @flow_control
       )
       super()
     end
+
+    # Supported Options
+    # FLOW_CONTROL - Flow control method NONE or RTSCTS. Defaults to NONE
+    def set_option(option_name, option_values)
+      super(option_name, option_values)
+      case option_name.upcase
+      when 'FLOW_CONTROL'
```

Comment:
I forgot we had OPTION. Since it can be anything it seems like it's impossible to document. Keep that in mind when you use it because I don't know how to support it with my config file editor.